### ISSUE TITLE
Pop-up window glitch (PHP)

### ISSUE THREAD

**Issue by Eugene-Melbourne** [apache/netbeans#7124]
### Apache NetBeans version

Apache NetBeans 21

### What happened

When you put a typing cursor in a special position the pop-up window with the code of the current function being shown - not requested and not helpful. It is very annoying.

### Language / Project Type / NetBeans Component

PHP project

### How to reproduce

This is a file example
[Foo.zip](https://github.com/apache/netbeans/files/14460497/Foo.zip)

It is important for the function to be bigger than the editor window size!

1) now put your typing cursor (not mouse cursor)  in the line 97 position 14
in other words after "{aaa}"

2) the same problem is visible when you put the cursor  in the line 97 position 13
in other words after "{aaa"

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 10 version 10.0 running on amd64; UTF-8; en_AU (nb)

### JDK

19.0.2; Java HotSpot(TM) 64-Bit Server VM 19.0.2+7-44

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by junichi11** [comment]
> It is very annoying.

Please don't write like this. You don't need to write this to resolve your issue, right? We won't create a bug intentionally.
That comment is not helpful at all. It just reduces developer motivation. Thanks for understanding.
